atest/acceptance/keywords/input_html5.robot

```diff
@@ -0,0 +1,64 @@
+*** Setting ***
+Test Setup        Go To Page "forms/html5_input_types.html"
+Resource          ../resource.robot
+
+*** Test Cases ***
+Input field type color
+    Input Text    id:color    \#72da19
+    ${value} =    Get Value    id:color
+    Log    ${value}    # Get value does not return the changed value
+
+Input field type date
+    Input Text    id:date    03/01/2019
+    ${value} =    Get Value    id:date
+    Should Be Equal As Strings    ${value}    2019-03-01
```

Comment:
Locale dependant test or just broken ?  Fails for 2019-01-03 != 2019-03-01

Reply:
Oh darn, should have seen that one coming. Should make a fix that the date is converted to same format.

Reply:
Created #1344 to address the problem.

Reply:
Fixed.